src/js/controller/controller.js

```diff
@@ -337,12 +338,26 @@ Object.assign(Controller.prototype, {
             if (model.get('playOnViewable')) {
                 if (viewable) {
                     _autoStart();
-                } else if (OS.mobile || (model.get('autoPause') && model.get('autoPause').viewability)) {
+                } else if (OS.mobile) {
                     _this.pause({ reason: 'autostart' });
                 }
             }
         }
 
+        if (_model.get('autoPause') && _model.get('autoPause').viewability) {
+            _model.set('pauseOnViewable', true);
```

Comment:
Why don't you do this in config.js so that it gets copied to the model? Do you ever set `pauseOnViewable` to false? Why don't you always add `autoPause` to the model, but set `viewability` to false when it's not set in the config. Then you can use `_model.autoPause.viewability` everywhere without additional `&&` checks.

Reply:
We don't currently have access to the model inside of config.js so I didn't want to add it unnecessarily. 

As for setting `viewability` to false - I'm confused. It should be defaulting to `true` if it's not set in the config. That's part of the AC.

Reply:
@waxidiotic Everything in the model is based on the config. It doesn't need access to the model.

1. When `autoPause` is undefined, add `{ autoPause: { viewability: false }` to the config (and thus the model.
2. When `autoPause` is defined add it and make sure viewability is the default correctly:
```js
{
  autoPause: {
    viewability: ('viewability' in autoPause) ? !!autoPause.viewability: true
}
```